Readme.md

```diff
@@ -1,21 +1,146 @@
-# Aragon Core
+# Aragon OSx Protool
 
-This workspace contains the Core foundation of the Aragon ecosystem - the human centered approach to DAO's.
+The Aragon OSx protocol is the foundation layer of the new Aragon stack. It allows users to create, manage, and customize DAOs in a way that is lean, adaptable, and secure.
 
-For more information on the individual packages, please read the respective `Readme.md`.
+Within this monorepo, you will be able to find 3 individual packages:
+- [Contracts](https://github.com/aragon/osx/tree/develop/packages/contracts): the Aragon OSx protocol contracts.
+- [Subgraph](https://github.com/aragon/osx/tree/develop/packages/subgraph): containing all code generating our subgraph and event indexing.
+- [Contract-ethers](https://github.com/aragon/osx/tree/develop/packages/contracts-ethers): containing the connection between the ethers package and our contracts.
+
+For more information on the individual packages, please read the respective `README.md`.
+
+## Contributing
+
+We'd love to hear what you think! If you want to build this with us, please find a detailed contribution guide in the `CONTRIBUTION_GUIDE.md` [file here](https://github.com/aragon/osx/blob/develop/CONTRIBUTION_GUIDE.md).
 
 ## Setup
 
-Start by running `yarn install` in the root.
+Start by running `yarn install` in the root in your terminal.
 
 ### Dependencies
 
 Since the repo is set up as yarn workspace, all the linking is done automatically.
 
+## How the Aragon OSx protocol works
+
+To review the contracts powering the Aragon OSx protocol, feel free to head to `packages/contracts`.
```

Comment:
Same as above

Reply:
Discord on this: https://discord.com/channels/672466989217873929/1077542094224236594/1078336009546252350